cvat/apps/quality_control/quality_reports.py

```diff
@@ -240,6 +241,9 @@ def from_dict(cls, d: dict):
             precision=np.asarray(d["precision"]),
             recall=np.asarray(d["recall"]),
             accuracy=np.asarray(d["accuracy"]),
+            # This field didn't exist at first, so it might not be present
+            # in old serialized instances.
```

Comment:
Could you update the existing tests as well or add a test that checks this value?

Reply:
Alright, updated.